### PR TITLE
added knife sheath

### PR DESCRIPTION
**Pull request by bleehu**
modified:   04_00_WeaponAttachments.txt

### CODE REVIEW

**Comment by bleehu** [comment]
At the request of Wildlovelies. The idea behind the long reload time is to avoid spamming knives.

File: Items/04_00_WeaponAttachments.txt [hunk 1]
```diff
@@ -515,6 +515,16 @@ draw your primary as a free action. Holstering your weapon is a free action.
...
     Strength Requirement:  -1
 
+== Concealed Throwing Knife Sheath $340 ==
+
+    This simple polylon band fits nicely around a wrist or ankle. Its easy to 
+conceal in a sleeve or boot. It holds up to three throwing knives in easy reach.
+
+    Dexterity requirement: 7
+    Knives thrown from the sheath may be drawn as a free action. Refilling the
+sheath takes twelve Actions.
```
Comment (Turtlelord26) [inline Items/04_00_WeaponAttachments.txt]: It still takes time to draw spare knives from wherever you have them stored. Three turns seems very excessive.
I would suggest targeting the refill time such that pulling knives from armor and putting them in (a pair of) sheaths takes a turn. Say 2 or 3 actions based on a reload DC.

If quick draw isn't rate limited for throwing weapons, that's a different problem with a different solution.

Comment (bleehu) [reply Items/04_00_WeaponAttachments.txt]: My concern is that this could be used to bypass existing quickdraw limits.

**Comment by bleehu** [comment]
@Turtlelord26 is that more reasonable?

**Comment by Turtlelord26** [comment]
Yes, but I think I'm going to push for the reload check to help out weapon skills. Sleight of hand is already super useful. Call it a nonblocking request.

**Review by Turtlelord26 — APPROVED** [review]

**Pull request merged**